Components/Maps/Maps.js

```diff
@@ -1,45 +1,54 @@
-import React, { useState, useEffect } from 'react';
-import { StyleSheet, View, Text, Dimensions } from 'react-native';
-import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
-import { Agenda } from 'react-native-calendars'
+import React, { useState } from 'react';
+import { StyleSheet, View, Dimensions } from 'react-native';
+import MapView from 'react-native-maps';
 import { Marker } from 'react-native-maps';
 
 
 export const Maps = ({ route }) => {
-  const { destination, startDate, endDate, name } = route.params
-  console.log('lat/long: ', destination);
-  console.log('dates: ', startDate, endDate);
+  const { dest, formattedMarkers } = route.params
+  const [ activities, setActivities ] = useState(formattedMarkers)
+  const [ startDate, setStartDate ] = useState(dest.destination.startDate)
+  const [ endDate, setEndDate ] = useState(dest.destination.endDate)
+  const [ savedActivities, setSavedActivities ] = useState([])
+  const [ region, setRegion ] = useState({
+    latitude: parseFloat(dest.destination.lat),
+    longitude: parseFloat(dest.destination.long),
+    latitudeDelta: 0.109,
+    longitudeDelta: 0.224
+  })
 
```

Comment:
This sets me up perfectly to build out the agenda functionality. Much appreciated